src/PlayerManager.cpp

```diff
@@ -548,6 +548,8 @@ void PlayerManager::setSpecialPlayer(CNSocket* sock, CNPacketData* data) {
         return; // ignore the malformed packet
 
     sP_CL2FE_GM_REQ_PC_SET_VALUE* setData = (sP_CL2FE_GM_REQ_PC_SET_VALUE*)data->buf;
+    PlayerView& plr = PlayerManager::players[sock];
```

Comment:
```suggestion
    Player *plr = PlayerManager::getPlayer(sock);
```
Use the `Player` struct directly unless you actually need the `PlayerView` structure as well.
Remember to update all the uses of `plr.plr` to `plr`